garak/probes/base.py

```diff
@@ -274,13 +274,19 @@ def probe(self, generator) -> Iterable[garak.attempt.Attempt]:
         attempts_todo: Iterable[garak.attempt.Attempt] = []
         prompts = list(self.prompts)
         lang = self.lang
-        prompts = self.langprovider.get_text(prompts)
+        # account for visual jailbreak until Turn/Conversation is supported
+        if isinstance(prompts[0], str):
+            prompts = self.langprovider.get_text(prompts)
+        else:
+            for prompt in prompts:
+                prompt["text"] = self.langprovider.get_text(prompt["text"])
         lang = self.langprovider.target_lang
         for seq, prompt in enumerate(prompts):
+            prompt_text = prompt
+            if not isinstance(prompt, str):
+                prompt_text = self.prompts[seq]["text"]
```

Comment:
guard against other prompt structures. maybe failure is wanted? maybe this is minor because turn will land soon? we have to live with this in v0.11.0 for a while, in any case.
```suggestion
            if isinstance(prompt, str):
                prompt_text = prompt
            elif "text" in self.prompts[seq]:
                prompt_text = self.prompts[seq]["text"]
```

Reply:
The other guard makes sense, the more I think about this change I am tempted to remove it completely as it may be more appropriate for the `note` to contain the serialization of the full original prompt not just text.